bin/inference/pycbc_inference

```diff
@@ -147,7 +148,31 @@ with ctx:
                 cp.get('sampler', 'checkpoint-signal')))
         # create an empty output file to keep condor happy
         open(opts.output_file, 'a').close()
+    
+    # Note: PyCBC's multi-ifo parser uses key:ifo for
+    # the injection file, even though we will use the same
+    # injection file for all detectors. This
+    # should be fixed in a future version of PyCBC. Once it is,
+    # update this. Until then, just use the first file.
+    if opts.injection_file:
+        injection_file = tuple(opts.injection_file.values())[0]  
+        # None if not set
+    else:
+        injection_file = None
 
+    # If injection placeholders for static params are set in config file, 
+    # insert injected parameters.
+    if injection_file \
+        and "injection" in dict(cp.items('static_params')).values():
+        inj = InjectionSet(injection_file)
+        for opt in cp.options(section='static_params'):
+            if cp.get('static_params', opt) == "injection" \
+                                and opt not in inj.table.fieldnames:
+                raise ValueError("Static param placeholder \"injection\" \
+                            has no counterpart in injection file.")
+            elif cp.get('static_params', opt) == "injection":
+                cp.set('static_params', opt, str(inj.table[opt][0]))
```

Comment:
Please see my comment in #2589 regarding `str(inj.table[opt][0])`. This is what causes the ringdown lmns parameter to be in the wrong format and it should be fixed here rather than having a complicated `format_lmns` function in the ringdown module because of this (@cdcapano has been disliking the shape that format_lmns was taking, and it is all because of this)